.travis.yml

```diff
@@ -44,7 +44,7 @@ before_script:
 
 script:
   - travis_wait ./gradlew check -Pkotlin.incremental=false --no-daemon
-  - ./gradlew runPrettyPrintersTests --no-daemon
+  - if [ $RUST_VERSION == $CURRENT_RUST_VERSION ]; then ./gradlew runPrettyPrintersTests --no-daemon; fi
```

Comment:
Looks like we want to run pretty printer tests for all rust versions except old one
Let's replace condition with `$RUST_VERSION != $OLD_RUST_VERSION` (not sure about correct bash syntax)